integration-tests/src/test/java/com/example/IntegrationTest.java

```diff
@@ -159,6 +161,12 @@
     assertThat(component.string()).isEqualTo("4");
   }
 
+  @Test public void implicitModules() {
+    ImplicitFinalModules component = backend.create(ImplicitFinalModules.class);
```

Comment:
Add `ignoreReflectBackend()` as the first statement so we can land this test before the fix.

Reply:
Done.